frontend/src/views/applications/SelectFacilityView.vue

```diff
@@ -34,6 +34,10 @@
         </v-col>
       </v-row>
     </div>
+
+    <div v-if="organization?.businessTypeCode === BUSINESS_TYPE_CODES.NON_PROFIT_SOCIETY" class="mb-6">
```

Comment:
this Not For Profit section needs to be above the checkbox and select facility dropdown

Reply:
done